airflow-core/src/airflow/models/dagbag.py

```diff
@@ -470,9 +470,10 @@ def _load_modules_from_zip(self, filepath, safe_mode):
 
     def _process_modules(self, filepath, mods, file_last_changed_on_disk):
         from airflow.models.dag import DAG  # Avoid circular import
+        from airflow.sdk import DAG as SDKDAG
```

Comment:
```suggestion
        from airflow.sdk import DAG as SdkDAG
```

Not sure whether it's better this way. `SDKDAG` is hard to read

Reply:
Hmm haha can be done 👍🏽

Reply:
I agree. You can also say `TaskSDKDag` :)

Reply:
like we do here:

https://github.com/apache/airflow/blob/fc63fb1d8a4d9851d5b8424859d8055dbcc25632/airflow-core/src/airflow/models/dag.py#L94

Reply:
We don't need both DAGs -- models.dag.DAG inherits from sdk's DAG, so we can just use one.

Reply:
> models.dag.DAG inherits from sdk's DAG

Right, let me try and do that. Its not needed as sdk DAG and models DAG will be covered by sdk DAG

Reply:
> models.dag.DAG inherits from sdk's DAG

Right, let me try and do that. Its not needed as sdk DAG and models DAG will be covered by sdk DAG

Reply:
This isn't needed anymore. Only importing from SDK as models DAG inherits it